app/src/main/java/co/condorlabs/customcomponents/customspinner/SpinnerFormField.kt

```diff
@@ -64,8 +63,10 @@ class SpinnerFormField(context: Context, attrs: AttributeSet) :
 
     override fun isValid(): ValidationResult {
         mSpinner?.let {
-            if (it.selectedItemPosition <= STATE_SPINNER_HINT_POSITION) {
-                return getErrorValidateResult()
+            if (mIsRequired) {
+                if (it.selectedItemPosition <= STATE_SPINNER_HINT_POSITION) {
```

Comment:
podría ser un solo condicional.
if ((it.selectedItemPosition <= STATE_SPINNER_HINT_POSITION) `&& mIsRequired`)